.github/workflows/blacklist.vac.txt

```diff
@@ -1,2 +1,3 @@
 ring_buffer_clean_up_unsat_test
-hash_table_foreach_unsat_test
\ No newline at end of file
+hash_table_foreach_unsat_test
+hash_table_create
```

Comment:
while this test is blacklisted in the CI, I assume that you did check for vacuity locally. If not, let me know so that we can dig deeper. Vacuity is important, so we need to do our best to make sure it passes.

Reply:
I ran and passed both vacuity and cex mod locally, but feel like they would take too long for CI;

Reply:
great